executor/aggfuncs/row_number_test.go

```diff
@@ -0,0 +1,18 @@
+package aggfuncs_test
```

Comment:
please add a license like others files.

Reply:
Can we write the test to window_func_test.go? like other window functions.